runtime/local/src/lib.rs

```diff
@@ -301,6 +301,8 @@ parameter_types! {
     pub const MinimumRemainingAmount: Balance = 1 * AST;
     pub const HistoryDepth: u32 = 14;
     pub const BonusEraDuration: u32 = 100;
+    pub const MaxUnlockingChunks: u32 = 32;
+    pub const UnbondingPeriod: u32 = 2;
```

Comment:
Please change `MaxUnlockingChunks` to equal 2.

It's not possible to have more chunks than what the `UnbondingPeriod` is.

I am aware that this is from Shibuya - it should be changed there too.

Reply:
done